docker/dev/angularjs/Dockerfile

```diff
@@ -1,10 +1,8 @@
-FROM node:9.4.0
-MAINTAINER CloudCV Team
+FROM node:12.2.0
 
-RUN apt-get update -qq && apt-get install -y build-essential git curl libfontconfig
+RUN apt-get update
+RUN apt-get install -y build-essential git curl libfontconfig
```

Comment:
Many places I see git curl but I think it is not used.

Reply:
Let's keep them for now. We'll remove them later if they won't be used.